enclave/core/sgx/enter.S

```diff
@@ -209,12 +246,14 @@ oe_enter:
     lfence
 
     // Clear the td_t.last_sp field (force oe_enter to calculate stack pointer)
-    movq $0, %fs:td_last_sp
+    movq $0, td_last_sp(%r11)
 
 .clear_enclave_registers:
 
     // Clear these so information will not be leaked to host
+    push %r11
```

Comment:
Leave a comment about saving %r11 and that it will be cleared after it is needed